### ISSUE TITLE
Replace misused anchor tags

### ISSUE THREAD

**Issue by Arnei** [opencast/opencast-admin-interface#100]
The new Admin UI uses `<a>` tags in many places where the arguably do not belong, e.g. the language select dropdown or the actions in the events details view. This is problematic from a usability and accessibility point-of-view, as they are not appropriately styled to accommodate for either of the two. For example, they are hardly keyboard navigable and will trip up screen reader users quite a lot. Thus, the `<a>` tags  should be replaced with more suitable tags.

I think using `<button>` tags makes sense in most places where this in an issue, although they need to be styled to maintain the current "look" of the Admin UI. Alternatively, we could build our own buttons from `<div>` tags, but that is probably more trouble than it is worth.
